hash/msx2_flop.xml

```diff
@@ -12436,6 +12466,34 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="saotomeb">
+		<description>Blue Wind (Japan)</description>
+		<year>1990</year>
+		<publisher>studio ANGEL</publisher>
+		<info name="alt_title" value="早乙女学園ブルーウィンド"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<info name="barcode" value="4988706200121"/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="blue wind - studio angel.dsk" size="737280" crc="61b95075" sha1="d0d609397008f5f0193d8b2fbc7b3ac62f2be3ae"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="saotomem">
+		<description>Saotome Military Academy of U.N. Space Force (Japan)</description>
+		<year>1988</year>
+		<publisher>studio ANGEL</publisher>
+		<info name="alt_title" value="国連宇宙軍士官学校 早乙女学園 入学案内"/>
```

Comment:
Is that a real official title?  If not, just use “Kokuren Uchuu-gun Shikan Gakkou Saotome Gakuen Nyuugaku Annai”.

Reply:
That is what it says on the title screen; it does not mention that title on the box though. Which one would be best to use in this case?

![0002](https://github.com/mamedev/mame/assets/9140441/26c01a1f-b410-416c-879f-e36f913746ea)

Reply:
In that case, “Saotome Military Academy of U.N. Space Force Nyuugaku Annai” will do.